slr/basic_app/templates/basic_app/frontend_submission.html

```diff
@@ -2,10 +2,84 @@
+                    <li><a class="navbar-link" href="{% url 'basic_app:frontend' %}">Front End</a></li>
+                    <li><a class="navbar-link" href="{% url 'basic_app:backend' %}">Back End</a></li>
+                    <li><a class="navbar-link" href="{% url 'basic_app:frontend_submission' %}">Front End Submission</a></li>
+                </ul>
+            </div>
+        </nav>
+          
+       <div class="banner">
+           <img class="banner-image" src="C:\Users\KAVYASRI DACHARLA\WSoC-TestRepo1\slr\static\basic_app\Logo-White-Header.png">
+       </div>
+       <h1><strong>FRONT END</strong></h1>
+       <p>Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+    <section class="parent">
+       <section class="child">
+        <h3>"WEB DEVELOPMENT"</h3>
+        <p><mark>Lorem ipsum</mark> dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. <strong>Quis aute</strong> iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+        <p><mark>Lorem ipsum</mark> dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur.<strong> Quis aute</strong> iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+        <p><mark>Lorem ipsum</mark> dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur.<strong> Quis aute </strong> reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+    </section>
+    <section class="child">
+        <img style="float: centre" ; padding: 15px 25px 10px 20px; src="C:\Users\KAVYASRI DACHARLA\WSoC-TestRepo1\slr\static\basic_app\doggo-bread.jpg" alt="Flowers in Chania" width="560" height="345">
```

Comment:
The path to the image is preferably relative whenever you are working with linking images, and the such.
Look into [this](https://www.w3schools.com/html/html_filepaths.asp) to understand how it works, and the potential benefits of working with relative file paths.

Reply:
Yes that's true relative path is better.Its my mistake

Reply:
Don't worry about it. I'm just pointing out better ways to solve issues.
Try to work with it, the next time you pick up a project. :)

Reply:
yeah sure.Thank you